bool is_started
bool is_stopped
bool is_running

